example/relational/README.md

```diff
@@ -43,8 +43,8 @@ The library does not enforce any special type for key modelling, anything castab
 
 ```go
 type Article struct {
-	Author   dynamo.IRI `dynamodbav:"prefix,omitempty"`
-	ID       dynamo.IRI `dynamodbav:"suffix,omitempty"`
+	Author   dynamo.IRI `dynamodbav:"prefix,omitempty"` => prefix == partion key? (why omitempty?)
+	ID       dynamo.IRI `dynamodbav:"suffix,omitempty"` => suffix == sort key?
```

Comment:
DynamoDB would write and empty string but it would fail partition key is not defined. Here, it just extra protection to ensure that application supply valid "object" identity to the library.